deploy/helm/kubecf/assets/operations/sequencing.yaml

```diff
@@ -0,0 +1,157 @@
+# We specify the new dependencies (I) and remove the old serialization
+# hints (II).
+#
+# I. Custom pod dependencies for startup sequencing
+##
+# ATTENTION: Annotations are strings. While the cf-operator's
+# controller will unmarshall the value as a json array later on,
+# specification must be done as a string using json syntax inside.
+##
+# Short form of the ops changes below.
+# Dependencies pointing to dependents.
+# Order here matches order of ops instructions.
+##
+# auctioneer <-- diego-api		(feature: not eirini)
+# scheduler  <-- api
+# cc-worker  <-- api
+# router     <-- routing-api <-- uaa	(feature: routing-api)
+#                tcp-router  <-- uaa	(feature: routing-api)
+#                log-cache   <-- uaa
+#                credhub     <-- uaa	(feature: credhub)
+#                diego-cell  <-- uaa	(feature: not eirini)
+#
+# asactors  <- asdatabase
+# asapi     <- asdatabase
+# asmetrics <- asdatabase
+# asnozzle  <- asdatabase
+##
+
+{{- if not .Values.features.eirini.enabled }}
+
+- type: replace
+  path: /instance_groups/name=auctioneer/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "diego-api" ]'
+
+{{- end }}
+
+- type: replace
+  path: /instance_groups/name=scheduler/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "api" ]'
+- type: replace
+  path: /instance_groups/name=cc-worker/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "api" ]'
+
+{{- if .Values.features.routing_api.enabled }}
+
+- type: replace
+  path: /instance_groups/name=router/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
```

Comment:
I don't think `router` is covered by `routing_api` is it?

Reply:
No. If I do not template it the `routing` will wait for the non-existent `routing-api`, would it not ?

Reply:
Oh yes of course, you're right.